### ISSUE TITLE
[a11y] VPN guarantee copy does not meet minimum color contrast threshold

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#15003]
## Description

The "30 money back guarantee" copy on the VPN landing page does not meet the minimum required color contrast threshold.

### Steps to reproduce

https://www.mozilla.org/en-US/products/vpn/

![image](https://github.com/user-attachments/assets/40d58ac0-7ca0-4229-a46f-27acca28ed8a)

### Expected result

The copy should meet the minimum required contrast

### Actual result

![image](https://github.com/user-attachments/assets/8078b90e-38e6-46aa-92df-1f52b9458b9b)

### Environment

N/A

**Comment by janbrasna** [comment]
For the occurrences on white background the `$color-moz-dark-green` (`#005e5e`) would pass both AA and AAA, however there's also an occurrence in plan comparison table on grey background, and that one would fail AAA for smaller text: https://webaim.org/resources/contrastchecker/?fcolor=005E5E&bcolor=EDEDF0

Also note this one has an icon in matching colour that needs to be updated along with the text:

> ![Screenshot 2024-08-23 at 23 53 33](https://github.com/user-attachments/assets/4f20fed4-9064-43d1-8cae-856f1cf8b9e2)

**Comment by alexgibson** [comment]
@janbrasna this looks like a good match to me. The icon is an SVG, so we can hopefully just edit the hex color to match :)

**Comment by alexgibson** [comment]
@janbrasna both yourself or anyone else is welcome to work o these, but yes I am not going to let them sit here as I’d like to start expanding coverage once we get the test job passing. Thanks for taking a look!

Right now I think just AA would suffice and at least allow the test job to be happy.

**Comment by janbrasna** [comment]
This 6.5:1 is not too far from AAA 7:1 for this small size, but passing that would mean using a custom colour outside from the Protocol palette so the PR #15038 only reaches AA for now, let's see at least what contrast ratio does aXe deem as sufficient.
